focus-ios/focus-ios-tests/XCUITest/URLValidationTest.swift

```diff
@@ -0,0 +1,30 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import XCTest
+
+class URLValidationTest: BaseTestCase {
+    override func setUp() {
+        super.setUp()
+        continueAfterFailure = true
+    }
+
+    let urlTypes = ["mozilla.org", "mozilla.org/", "https://mozilla.org", "mozilla.org/en", "mozilla.org/en-",
+                    "mozilla.org/en-US", "https://mozilla.org/", "https://mozilla.org/en", "https://mozilla.org/en-US"]
+
+    func testDifferentURLTypes() {
```

Comment:
Can we add this test to Test Rail (if not yet) and have the reference here please?

Reply:
Updated

Reply:
Thank you!